iosxr_test.py

```diff
@@ -63,34 +66,41 @@ def check_result(result, success_message):
 
 def bringup_vagrant():
     '''
-    Bring up a vagrant box.
-
-    Clean up ssh keys from the generation of the virtualbox.
-
-    Use pxssh to fascilitate the ssh process.
+    Bring up a vagrant box and test the ssh connection.
     '''
+
     # Clean up Vagrantfile
     try:
         os.remove('Vagrantfile')
     except OSError:
         pass
 
+    global iosxr_port
+    global linux_port
+
+    # Use vagrant to init, add and bring up the inputted Vagrant VirtualBox
     logger.debug("Bringing up '%s'..." % input_box)
 
-    run(['vagrant', 'init', 'XRv64-test'])  # Single node for now, in future could bring up two nodes and do more testing
-    run(['vagrant', 'box', 'add', '--name', 'XRv64-test', input_box, '--force'])
+    output = run(['vagrant', 'init', 'XRv64-test'])
+    logger.debug(output)
```

Comment:
I think we'd talked before about how this is not ideal, as it won't generate any output while the subprocess runs, then dump all output at once only after the subprocess terminates. (Plus, if the subprocess errors out, will we even reach this line?)